File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/ScarletShroud.java
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances;

import it.paranoidsquirrels.idleguildmaster.R;
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.MediumArmor;

/* JADX INFO: loaded from: classes3.dex */
public class ScarletShroud extends MediumArmor {
    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.Item
    protected void configureProperties() {
        this.idName = R.string.armor_medium_scarlet_shroud_name;
        this.idDescription = R.string.armor_medium_scarlet_shroud_description;
        this.idEffect = R.string.armor_medium_scarlet_shroud_effect;
        this.idImage = R.drawable.scarlet_shroud;
        this.price = 135000L;
        this.maxHp = 300;
        this.constitution = 50;
        this.dexterity = 40;
        this.criticalChance = 0.21d;
        this.criticalDamage = 0.21d;
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/ScarletShroud.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances

import it.paranoidsquirrels.idleguildmaster.R
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.MediumArmor

class ScarletShroud : MediumArmor() {
    override fun configureProperties() {
        idName = R.string.armor_medium_scarlet_shroud_name
        idDescription = R.string.armor_medium_scarlet_shroud_description
        idEffect = R.string.armor_medium_scarlet_shroud_effect
        idImage = R.drawable.scarlet_shroud
        price = 135000L
        maxHp = 300
        constitution = 50
        dexterity = 40
        criticalChance = 0.21
        criticalDamage = 0.21
    }
}
